### ISSUE TITLE
Visualize if a day is a public holiday  (Norwegian)

### ISSUE THREAD

**Issue by trondoe** [Puzzlepart/did#1064]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
Adding a red colour to a date that is a public holiday. See https://webapi.no/api/v1/holidays/2022 for API that can be used to get dates and names for public holidays in Norway.

Since public holidays are static for each year it makes sense to reduce the number of calls to the API to an absolute minimum. Maybe store the data in Did after making a call to the holidays api?

### Description
* (What) This feature makes it easier to spot days that are public holidays in the week view of Did.
* (Why) We should add this feature to make it more obvious to the user why a day in Did has 0 hours registered (given that it is a public holiday). The feature also opens up the possibility of presenting "available hours each month given 8 hours workdays minus public holidays" to the users. This improves the product in so many ways that even Chuck Norris approves this feature. 

![image](https://user-images.githubusercontent.com/8970087/202710055-3c13067b-92aa-4bdc-9f4d-2adb2981e03d.png)
